src/AnnouncementBubble/DeletedAnnouncementBubble.tsx

```diff
@@ -12,8 +12,9 @@ function cssClass(element: string) {
 
 export interface Props {
   bubbleType: "deleted";
-  theme?: MessagingTheme;
+
```

Comment:
why space?

Reply:
I have no idea if this convention actually makes sense to anyone else, but I put `bubbleType` at top because it's essentially 'preset' for each bubble version. i.e. There's only valid value of it for any given permutation of bubble and type.

The rest of the props I separated with a space and alphabetized, then had a separate space if there was a "temporary" prop, per the work on making everything accept translations for usage in fampo, until translations live in Dewey.

Curious what you think of this!

Reply:
ahhh that does make sense!